Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM qazbnm456/fbctf_base:latest
```

Comment:
The image "qazbnm456/fbctf_base" is just doing pre-install some softwares to make building process more quickly, and its source code is also available on github. If you want to replace it, use phusion/baseimage:latest instead. The reason is that qazbnm456/fbctf_base is built from that image, and some provided scripts are also dependent on that , too.